crates/telio-traversal/src/batcher.rs

```diff
@@ -242,23 +284,371 @@ where
     }
 }
 
-/// Tests provide near-perfect immediate sleeps due to how tokio runtime acts when it's paused(basically sleeps are resolved immediately)
 #[cfg(test)]
 mod tests {
+    use crate::batcher::{Batcher, BatcherError, BatcherTrait};
+    use std::{sync::Arc, time::Duration};
```

Comment:
Why are we using both `std::time:Duration` and `tokio::time::Duration`?

Reply:
Good point, it was simply overwritten by tokio's, removed that